jsapp/xlform/src/model.row.coffee

```diff
@@ -273,6 +290,7 @@ module.exports = do ->
         ctx.warnings.push "Score choices list not set"
         @_scoreChoices = @getSurvey().choices.add(name: txtid())
       ``
+      return
```

Comment:
Same question here - shouldn't we remove the backtics? Maybe two backticks were being used as implicit return?